src/crosschain/trade/lifiTrade.ts

```
@@ -0,0 +1,174 @@
+    public async init() {
+        const apiKeys = this.symbiosis.lifiConfig.apiKeys
+        if (apiKeys.length === 0) {
+            throw new LifiTradeError('Missing LiFi API key')
+        }
+        const apiKey = apiKeys[Math.floor(Math.random() * apiKeys.length)]
+
+        const chainId = this.tokenAmountIn.token.chainId
+
+        const fromToken = this.tokenAmountIn.token.isNative
+            ? NATIVE_TOKEN_ADDRESS
+            : this.tokenAmountIn.token.address
```

ESLint

[error] Replace `⏎············?·NATIVE_TOKEN_ADDRESS⏎···········` with `·?·NATIVE_TOKEN_ADDRESS`

(prettier/prettier)